netfields/fields.py

```diff
@@ -74,6 +74,7 @@ def deconstruct(self):
 
 
 class InetAddressField(with_metaclass(models.SubfieldBase, _NetAddressField)):
+    short_name = "IP"
```

Comment:
Since these are only used by the REST framework fields, can we perhaps define these there?